netty/src/test/java/io/grpc/netty/ProtocolNegotiatorsTest.java

```diff
@@ -246,7 +246,7 @@ public void tlsHandler_failsOnNullEngine() throws Exception {
 
   @Test
   public void tlsHandler_handlerAddedAddsSslHandler() throws Exception {
-    ChannelHandler handler = new ServerTlsHandler(grpcHandler, sslContext);
+    ChannelHandler handler = new ServerTlsHandler(grpcHandler, sslContext, null);
```

Comment:
For all `null` or literal values passed as arguments, it is recommended to use a comment like `executorPool` as the parameter name. For all call sites (lines 258, 279, ...)

Reply:
I think for tests we generally don't care.